aquacomputer_d5next.c

```diff
@@ -371,6 +401,8 @@ static umode_t aqc_is_visible(const void *data, enum hwmon_sensor_types type, u3
 	case hwmon_temp:
 		if (channel < priv->num_temp_sensors)
 			return 0444;
+		if (channel < priv->num_virtual_temp_sensors)
```

Comment:
should be `if (channel < priv->num_temp_sensors + priv->num_virtual_temp_sensors)` to accommodate for all virtual sensors